dcos/dcos-ui
# DCOS-40469: update http service

## Testing


1. Install new module `$ npm i`
2. Check out the corresponding plugin branch
3. Check the UI and see if this change introduced a bug in the areas touched by the update.


## Trade-offs



- Decided to keep the MetronomeClient open, so that it consumers can decide if they want to get the status code

## Dependencies



None

## Screenshots



No visual change
